pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
So for me, running the "sw_vers" command without the prepended "/usr/bin/" works perfectly alright. I want to see if NixOS contains this already